examples/cpp/pyperf/PyPerfNativeStackTrace.cc

```diff
@@ -200,5 +226,36 @@ bool NativeStackTrace::error_occured() const {
   return error_occurred;
 }
 
+void NativeStackTrace::prune_dead_pid(uint32_t dead_pid) {
+  auto it = procSymbolsCache.find(dead_pid);
+  if (it != procSymbolsCache.end()) {
+    procSymbolsCache.erase(it);
+  }
+}
+
+void NativeStackTrace::enable_dso_reporting() {
+  NativeStackTrace::insert_dso_name = true;
+}
+
+ProcSyms* NativeStackTrace::get_proc_symbols(uint32_t pid) {
+  static struct bcc_symbol_option symbol_options = {
```

Comment:
```suggestion
  static const struct bcc_symbol_option symbol_options = {
```

Reply:
Or actually, it seems that this function doesn't get `const`. Oh well

Reply:
That would require changing c-tor signature for `ProcSyms`. This could be a part of a larger effort, once we decide we decide improvements are necessary in baseline code.

Reply:
No no, not required. If the function has a bad signature then that's another problem.